keyboards/planck/keymaps/samuel/keymap.c

```diff
@@ -0,0 +1,298 @@
+#include "planck.h"
+#include "action_layer.h"
```

Comment:
Could you remove this additional include?  

```suggestion
```

They get included automatically, at various points in the built process, so they are redundant and not necessary here.